Makefile

```diff
@@ -2,22 +2,22 @@
 SHELL := /bin/bash
 
 # versions at  https://github.com/kubernetes-sigs/controller-tools/releases
-CONTROLLER_GEN_VERSION = v0.10.0
+CONTROLLER_GEN_VERSION = v0.12.0
 
 # GO_VERSION refers to the version of Golang to be downloaded when running dockerized version
-GO_VERSION = 1.19
+GO_VERSION = 1.20
 
 # ENVTEST_K8S_VERSION refers to the version of kubebuilder assets to be downloaded by envtest binary.
 ENVTEST_K8S_VERSION = 1.23
 
 # versions at https://github.com/operator-framework/operator-sdk/releases
-OPERATOR_SDK_VERSION = v1.25.3
+OPERATOR_SDK_VERSION = v1.28.1
```

Comment:
Can we update the scorecard tests images in https://github.com/medik8s/self-node-remediation/tree/main/config/scorecard/patches as part of updating the OPERATOR_SDK_VERSION? See old PR for reference #54